app/models/application.rb

```diff
@@ -30,7 +30,7 @@ class Application < ApplicationRecord
   scope :unsynced, -> { where(ecf_id: nil) }
   scope :expired_applications, -> { where(lead_provider_approval_status: "rejected").where("created_at < ?", cut_off_date_for_expired_applications) }
   scope :active_applications, -> { where.not(id: expired_applications) }
-  scope :accepted, -> { where(lead_provider_approval_status: "accepted") }
+  scope :accepted, -> { where(lead_provider_approval_status: "accepted").where.not(accepted_at: nil) }
```

Comment:
I think we should not look into `accepted_at` and ignore it as the determinate is the lead provider approval status, if we get to a state it's not correct its a data issue which hopefully we won't have in production. Let me know you're thoughts :)